### PR TITLE
Fix consul_sd_config to follow types convention

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: docs/configuration/configuration.md [hunk 1]
```diff
@@ -359,9 +359,9 @@ services:
 tags:
   [ - <string> ]
 
-# Node metadata used to filter nodes for a given service.
+# Node metadata pairs used to filter nodes for a given service.
```
Comment (brian-brazil) [inline docs/configuration/configuration.md:362]: "names and values" maybe?

Comment (roidelapluie) [reply docs/configuration/configuration.md:362]: Here is what consul says:

```
Metadata to filter nodes with the given key=value pairs. This flag may be specified multiple times to filter on multiple sources of metadata.
```

Thet do not speak about names but keys/values.

Comment (roidelapluie) [reply docs/configuration/configuration.md:362]: Should I just put

```
Metadata to filter nodes with the given key=value pairs for a given service
```

as description?

Comment (brian-brazil) [reply docs/configuration/configuration.md:362]: Okay for that bit, but the grammar feels off. How about:

Node metadata key/value pairs to filter nodes for a given service

Comment (roidelapluie) [reply docs/configuration/configuration.md]: fixed

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**